articles/digital-twins/concepts-models.md

```diff
@@ -186,7 +186,7 @@ For a comprehensive list of the fields that may appear as part of a component, s
 
 Here's a basic example of a component on a DTDL model. This example shows a Room model that makes use of a thermostat model as a component.
 
-:::code language="json" source="~/digital-twins-docs-samples-getting-started/models/advanced-home-example/IRoom.json" highlight="15-19, 28-41":::
+:::code language="json" source="~/digital-twins-docs-samples-getting-started/models/advanced-home-example/IRoom.json" highlight="18-22, 31-46":::
```

Comment:
Requested change: `highlight="18-22, 31-47"`

Explanation: If the highlight includes the opening bracket, it would be good to also include the closing bracket.
![image](https://github.com/user-attachments/assets/e55440e3-bb1b-40c2-bd58-850fc0dc5f0f)